include/groonga/cache.h

```diff
@@ -42,7 +44,8 @@ GRN_API const char *grn_get_default_cache_base_path(void);
  *
  * Persistent Cache:
  * - The same cache is shared across multiple processes.
- * - Persistent cache is slightly slower than memory cache due to filesystem I/O.
+ * - Persistent cache is slightly slower than memory cache due to filesystem
+ * I/O.
```

Comment:
Is this rendered correctly?

Do we need to format this something like the following?

```c
 * - Persistent ...
 *   I/O.
```

Reply:
The document was rendered as expected.
But it was easier to see if the indentation was aligned, so it was aligned.